client/generated_volume.go

```diff
@@ -120,6 +120,8 @@ type VolumeOperations interface {
 	ActionSnapshotPurge(*Volume) (*Volume, error)
 
 	ActionSnapshotRevert(*Volume, *SnapshotInput) (*Snapshot, error)
+
+	ActionUpdateDataLocality(*Volume, *UpdateDataLocalityInput) (*Volume, error)
```

Comment:
I don't think we need this in this PR? If we still want the action, we can update it in another PR.

Reply:
Thank you. Fixed